plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/Main.java

```diff
@@ -0,0 +1,39 @@
+package io.jenkins.tools.pluginmanager.cli;
+
+import io.jenkins.tools.pluginmanager.cli.CliOptions;
+import io.jenkins.tools.pluginmanager.impl.PluginManager;
+import io.jenkins.tools.pluginmanager.config.Config;
+
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+
+import java.io.IOException;
+
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        CliOptions options = new CliOptions();
+        CmdLineParser parser = new CmdLineParser(options);
+
+        try {
+            parser.parseArgument(args);
+        } catch (CmdLineException e) {
+            parser.printUsage(System.out);
+            throw new IOException("Failed to read command-line arguments", e);
+        }
+
+        Config cfg = new Config();
+
+        cfg.setPluginTxt(options.getPluginTxt());
+        cfg.setPluginDir(options.getPluginDir());
+        cfg.setShowWarnings(options.hasShowWarnings());
+        cfg.setShowAllWarnings(options.hasShowAllWarnings());
+
+        String[] plugins = options.getPlugins();
```

Comment:
It doesn't look like you're doing anything with the list of plugins that can be passed into the CLI here.  The Config should make sure that either (or both) the plugin.txt file and the ones passed in on the command line are respected.

Reply:
Fixed in 7ca3d5d987f6c60617f5f95994574849fb17f170.